File: EVA2/c++/Eva2_3_Nasted_If2.cpp
#include <iostream>
#include <sstream>
#include <string>
#include <cstdlib>
#include <cmath>

using namespace std;

// Headers
string toString (double);
int toInt (string);
double toDouble (string);

int main() {
    cout << "¿Cuál es tu calificación?" << endl;
    int iCalif;
    
    cin >> iCalif;
    if (iCalif >= 90 && iCalif <= 100) {
        cout << "Tienes una A" << endl;
    } else {
        if (iCalif >= 80 && iCalif <= 89) {
            cout << "Tienes una B" << endl;
        } else {
            if (iCalif >= 70 && iCalif <= 79) {
                cout << "Tienes una C" << endl;
            } else {
                if (iCalif >= 60 && iCalif <= 69) {
                    cout << "Tienes una D" << endl;
                } else {
                    if (iCalif >= 50 && iCalif <= 59) {
                        cout << "Tienes una F" << endl;
                    } else {
                        cout << "numero no valido" << endl;
                    }
                }
            }
        }
    }
    return 0;
}

// The following implements type conversion functions.
string toString (double value) { //int also
    stringstream temp;
    temp << value;
    return temp.str();
}

int toInt (string text) {
    return atoi(text.c_str());
}

double toDouble (string text) {
    return atof(text.c_str());
}

File: EVA2/python/Eva2_3_Nasted_If2.py
print("¿Cuál es tu calificación?")
iCalif = int(input())
if iCalif >= 90 and iCalif <= 100:
    print("Tienes una A")
else:
    if iCalif >= 80 and iCalif <= 89:
        print("Tienes una B")
    else:
        if iCalif >= 70 and iCalif <= 79:
            print("Tienes una C")
        else:
            if iCalif >= 60 and iCalif <= 69:
                print("Tienes una D")
            else:
                if iCalif >= 50 and iCalif <= 59:
                    print("Tienes una F")
                else:
                    print("numero no valido")
